### PR TITLE
Refactor translator().reverse().convert(resource).asNode()

### PR DESCRIPTION
**Pull request by awoods**
Resolves: https://jira.duraspace.org/browse/FCREPO-1932

### CODE REVIEW

File: fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraBaseResource.java [hunk 1]
```diff
@@ -56,6 +57,10 @@
         return idTranslator;
     }
 
+    protected Node asNode(final FedoraResource resource) {
```
Comment (ajs6f) [inline fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraBaseResource.java]: Obviously, this is a good shortcut. Maybe a quick JavaDoc here just for those unfamiliar with the codebase?

Comment (awoods) [inline fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraBaseResource.java]: Javadoc added...
Please do not commit this PR without first squashing these two commits. I would be happy to do the squash if you prefer.

**Comment by ajs6f** [comment]
You squash, I'll merge. Mental note: we should probably revisit the Guava `Converter` abstraction on which we built the id translation classes. Post Java 8, there might be better ways to do this stuff.

**Comment by awoods** [comment]
squashed

**Pull request merged**